progressSearch.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Profile</title>
+	<link rel="stylesheet" type="text/css" href="usersearch.css">
+</head>
+<body>
+	<!--USER TYPES IN THEIR MEMBERSHIP ID IN THE SEARCH FIELD AND IT WILL RETIRIEVE THEIR WORKOUT HISTORY DETAILS ON THE NEXT PAGE-->
+	<h1> Type in your Membership ID:</h1>
```

Comment:
Page functioning correctly with search field displaying for users to enter their membership ID